stack/stack.go

```diff
@@ -92,7 +93,9 @@ func fetchYAML(address *url.URL) ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
+	req.Header.Set("User-Agent", fmt.Sprintf("%s/%s", "FaaS-CLI", version.VersionStr()))
```

Comment:
`"FaaS-CLI"` could be a const in the version package, or a function:

`version.UserAgent()`

probably `OpenFaaS CLI` vs `FaaS-CLI`